<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>969</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttom_1_wire_cross_section">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Сечение провода</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttom_2_lenght_jumper_battery">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Длина перемячки между АКБ</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_3_lenght_wire_to_positive">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Длина провода от первого АКБ до автомата</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttom_4_lenght_wire_to_negative">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Длина провода от последнего АКБ до автомата</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttom_5_battery_voltage">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Напряжение АКБ</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttom_6_battery_internal_resistance">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>570</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Внутреннее сопротивление АКБ</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttom_7_battery_quantity">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>670</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Кол-во АКБ</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttom_8_current_consumer">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>770</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Ток потребителя</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>691</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>NoAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Калькулятор значений АКБ</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_from_user_1_wire_cross_section">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_from_user_2_lenght_jumper_battery">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_from_user_3_lenght_wire_to_positive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_from_user_4_lenght_wire_to_negative">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_from_user_5_battery_voltage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_from_user_6_battery_internal_resistance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_from_user_7_battery_quantity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_from_user_8_current_consumer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_to_user_1_result">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>70</y>
      <width>380</width>
      <height>760</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>850</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Расчитать</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="output_value_1_wire_cross_section">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_value_2_lenght_jumper_battery">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_value_3_lenght_wire_to_positive">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_value_4_lenght_wire_to_negative">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_value_5_battery_voltage">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_value_6_battery_internal_resistance">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>570</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_value_7_battery_quantity">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>670</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_value_8_current_consumer">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>770</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>buttom_1_wire_cross_section</zorder>
   <zorder>buttom_2_lenght_jumper_battery</zorder>
   <zorder>button_3_lenght_wire_to_positive</zorder>
   <zorder>buttom_4_lenght_wire_to_negative</zorder>
   <zorder>buttom_5_battery_voltage</zorder>
   <zorder>buttom_7_battery_quantity</zorder>
   <zorder>buttom_8_current_consumer</zorder>
   <zorder>buttom_6_battery_internal_resistance</zorder>
   <zorder>label</zorder>
   <zorder>input_from_user_1_wire_cross_section</zorder>
   <zorder>input_from_user_2_lenght_jumper_battery</zorder>
   <zorder>input_from_user_3_lenght_wire_to_positive</zorder>
   <zorder>input_from_user_4_lenght_wire_to_negative</zorder>
   <zorder>input_from_user_5_battery_voltage</zorder>
   <zorder>input_from_user_6_battery_internal_resistance</zorder>
   <zorder>input_from_user_7_battery_quantity</zorder>
   <zorder>input_from_user_8_current_consumer</zorder>
   <zorder>output_to_user_1_result</zorder>
   <zorder>start_button</zorder>
   <zorder>output_value_1_wire_cross_section</zorder>
   <zorder>output_value_2_lenght_jumper_battery</zorder>
   <zorder>output_value_3_lenght_wire_to_positive</zorder>
   <zorder>output_value_4_lenght_wire_to_negative</zorder>
   <zorder>output_value_5_battery_voltage</zorder>
   <zorder>output_value_6_battery_internal_resistance</zorder>
   <zorder>output_value_7_battery_quantity</zorder>
   <zorder>output_value_8_current_consumer</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
